docs/transport/braginskii.rst

```diff
@@ -47,3 +52,14 @@ Viscosities
 These include:
 ion_viscosity
 electron_viscosity
+
+Reference/API
+=============
+
+.. automodapi:: plasmapy.physics.transport.braginskii
+    :skip: Coulomb_logarithm
+    :skip: Hall_parameter
+    :skip: collision_rate_electron_ion
+    :skip: collision_rate_ion_ion
+    :no-heading:
+    :no-main-docstr:
```

Comment:
... but you probably don't want to duplicate what you wrote up there, so I am disabling duplication of the main docstring from braginskii.py, which currently just states
```python
"""Functions to calculate classical transport coefficients."""
```

I am not sure about any of this, so please feel free to adjust!